bundles/org.openhab.binding.evcc/README.md

```diff
@@ -42,8 +42,66 @@ Default value for _refreshInterval_ is 30 seconds.
 
 ## Thing(s) Configuration
 
-The Things don't have a configuration. They will be set up automatically when a bridge has been configured.
-You can add them via file configuration or in the openHAB UI.
+Things will be set up automatically when a bridge has been configured and will appear in your inbox.
+It is not possible to add things manually in the UI.
+Nevertheless, you can add them manually via things file.
+
+### Thing Site
+
+The Site Thing represents the overall site data of your evcc instance.
+No configuration parameters are required.
+
+### Thing Battery
+
+The Battery Thing represents a battery configured in your evcc instance.
+Needs the index of the battery as configuration parameter.
+
+### Thing PV
+
+The PV Thing represents a photovoltaic system configured in your evcc instance.
+Needs the index of the photovoltaic system as configuration parameter.
+
+### Thing Loadpoint
+
+The Loadpoint Thing represents a loadpoint configured in your evcc instance.
+Needs the index of the loadpoint as configuration parameter.
+
+### Thing Vehicle
+
+The Vehicle Thing represents a vehicle configured in your evcc instance.
+Needs the database id of the vehicle as configuration parameter.
```

Comment:
For site, battery, pv, loadpoint and vehicle, the description is already provided in `## Supported Things`

As all things except the bridge needs the database id, i think that line can be added to `## Thing(s) Configuration`.  This way these sub paragraphs can be omitted. This makes the documentation a little less verbose.